lib/pbench/server/indexing_tarballs.py

```diff
@@ -1,5 +1,6 @@
 """Initialising Indexing class"""
 
+import json
 import os
 import glob
 import signal
```

Comment:
Alphabetize?

Reply:
Ha; I just added `json` in order and didn't really even look at the others. I'm not sure now whether I assumed they were already ordered, or just didn't care. ;-)

Reply:
"Hmmm...does `j` come before `o` or after `g`??  Well, normally the answer would be 'yes', but that's not useful here...."  🤣